draft-ietf-httpbis-digest-headers.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Digest Fields
+title: Structured Digest Fields
```

Comment:
I would keep that out of the title; it's a syntax detail.

Reply:
Fine by me!

Reply:
Agreed, doesn't need to be in the title.

Reply:
```suggestion
title: Digest Fields
```